docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -195,12 +195,13 @@ metadata:
   name: nova-compute-extraconfig
   namespace: openstack
 spec:
-  label: nova.compute.extraconfig
-  configMaps:
-    - nova-compute-extraconfig
-  secrets:
-    - nova-cell1-compute-config
-    - nova-migration-ssh-key
+  dataSources:
```

Comment:
I kept nova-compute-extraconfig service, to keep the simmetry with the ceph test

Reply:
well i would prefer to chnage the ceph test to use nova-extra-config insetad